go/cmd/pfacct/radius.go

```diff
@@ -134,25 +143,34 @@ func (h *PfAcct) handleAccountingRequest(rr radiusRequest) {
 	timestamp = timestamp.Truncate(h.TimeDuration)
 	node_id := mac.NodeId(uint16(switchInfo.TenantId))
 	unique_session_id := h.accountingUniqueSessionId(r)
-	if err := h.InsertBandwidthAccounting(
-		node_id,
-		switchInfo.TenantId,
-		mac.String(),
-		unique_session_id,
-		timestamp,
-		in_bytes,
-		out_bytes,
-	); err != nil {
-		logError(ctx, "InsertBandwidthAccounting: "+err.Error())
-	}
-
-	if status == rfc2866.AcctStatusType_Value_Stop {
-		h.CloseSession(node_id, unique_session_id)
-	}
+	node_acct_info := acct_info{node_id: node_id,
+		tenant_id:      switchInfo.TenantId,
+		mac:            mac.String(),
+		unique_session: unique_session_id,
+		bucket:         timestamp,
+		in_bytes:       in_bytes,
+		out_bytes:      out_bytes,
+	}
+	go func() {
+		var ok bool
+		select {
+		case h.JobChan <- node_acct_info:
+			ok = true
+		default:
+			ok = false
+		}
+		if !ok {
+			logError(ctx, "handleAccountingRequest: accouting queue - max capacity reached")
```

Comment:
I'm guessing you should return here, otherwise you'll call the code below which substracts time balance and all the other stuff. I would also be good to not reply to the packet so that the controller sends a new request

Reply:
Also, on top of my comment above, just a small typo to fix in the log line

```suggestion
			logError(ctx, "handleAccountingRequest: accounting queue - max capacity reached")
```